lib/keyboard.js

```diff
@@ -0,0 +1,134 @@
+(function(T) {
+    "use strict";
+
+    if (T.envtype !== "browser") {
+        return;
+    }
+
+    var fn = T.fn;
+    var instance = null;
+
+    function KeyboardListener(_args) {
```

Comment:
Do we need a new keyboard listener? We have a global one already, can that be extended for the purpose here, or is it important that this only listen for keys pressed while the playground is focused?